src/arch/xtensa/drivers/timer.c

```diff
@@ -16,7 +16,7 @@ struct timer_data {
 	void *arg2;
 };
 
-static struct timer_data xtimer[ARCH_TIMER_COUNT] = {};
+static struct timer_data xtimer[ARCH_TIMER_COUNT];
```

Comment:
I think it depends whether the RTOS zeroes out the bss or not, but we can assume that every arch will do that.

Reply:
@tlauda If BSS isn't zeroed, I think the system isn't C-compliant and we're in a trouble.